packages/navigation/src/scss/_local-nav.scss

```diff
@@ -0,0 +1,108 @@
+@use "@uqds/core/src/scss/global" as core;
+@use "@uqds/icon/src/scss/global" as icon;
+@use "global" as *;
+
+.uq-sr-only {
```

Comment:
Such a shame this is the solution to extending meaning on a link for screen readers.

I had a quick look at solutions: 
- https://usability.yale.edu/web-accessibility/articles/links
- https://www.visionaustralia.org/seda/business-consulting/digital-access/blog/how-to-make-read-more-links-accessible

The only other way to go here would have been aria-label, repeating the full text. 

I'll try to get some additional testing in, but won't hold this up, as the solution appears to be the most common recommended.